ci/reload-env.sh

```diff
@@ -91,3 +91,5 @@ function retry {
 alias pip="retry pip"
 shopt -s expand_aliases
 set -e
+# https://github.com/pypa/setuptools/issues/3504
+export SETUPTOOLS_ENABLE_FEATURES="legacy-editable"
```

Comment:
This should be reverted, setuptools has been fixed in master

Reply:
Thanks. I removed this commit and merged latest master.